### PR TITLE
Mayhem integration initial

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone. Use `ADD` instead.

**Review by xansec — CHANGES_REQUESTED** [review]
Please use a generic project name.  Please make the requested changes.

**Comment by code-terror** [comment]
@xansec Done!

**Comment by xansec** [comment]
LGTM!

**Review by xansec — APPROVED** [review]

**Pull request merged**